src/index.html.ejs

```diff
@@ -17,7 +17,7 @@
 <body>
   <div class="header">
     <!-- FIXME images alias isn't getting picked up / translated by webpack -->
-    <img class="logo" src="~images/logo.png" />
+    <img class="logo" src="<%= require('images/logo.png') %>" />
```

Comment:
I couldn't get it to understand the `~` alias, even with some fiddling around with the webpack config :man_shrugging: